javascript/input-and-string/js-inputs-and-strings_multiple-inputs/js/index.js

```diff
@@ -14,3 +14,20 @@ Implement the following functionality:
 const firstInput = document.querySelector("[data-js=first-input]");
 const secondInput = document.querySelector("[data-js=second-input]");
 const button = document.querySelector("[data-js=button]");
+
+const copyValue = () => {
+  secondInput.value = firstInput.value;
+};
+
+const copyUppercaseValue = () => {
+  secondInput.value = firstInput.value.toUpperCase();
+};
+
+const switchValues = () => {
+  [firstInput.value, secondInput.value] = [secondInput.value, firstInput.value];
```

Comment:
i don't know this thing, maybe works but i cannot see it as all functions are on at a time so work simultaneously. You need to check (or have already) by yourself if it works if only this one is on.